src/utils/date.ts

```diff
@@ -1,39 +1,65 @@
+import {Language, TranslateFunction, dictionary} from '@atb/translations';
 import {
+  FormatOptions,
+  Locale,
   addHours,
   differenceInCalendarDays,
   differenceInMinutes,
   differenceInSeconds,
-  format,
-  getHours,
-  getMinutes,
+  format as fnsFormat,
   isAfter as fnsIsAfter,
   isBefore as fnsIsBefore,
+  getHours,
+  getMinutes,
+  getSeconds,
   isPast,
   isSameDay,
   isSameYear,
   isToday,
   isWithinInterval,
-  Locale,
   parse,
   parseISO,
   set,
-  getSeconds,
 } from 'date-fns';
-import en from 'date-fns/locale/en-GB';
-import nb from 'date-fns/locale/nb';
+import {
+  FormatOptionsWithTZ,
+  formatInTimeZone,
+  fromZonedTime,
+} from 'date-fns-tz';
+import {enGB as en, nb} from 'date-fns/locale';
 import humanizeDuration from 'humanize-duration';
 
+export {daysInWeek} from 'date-fns/constants';
+
 import {
   parse as parseIso8601Duration,
   toSeconds as toSecondsIso8601Duration,
 } from 'iso8601-duration';
 
-import {
-  DEFAULT_LANGUAGE,
-  Language,
-  TranslateFunction,
-  dictionary,
-} from '@atb/translations';
+const CET = 'Europe/Oslo';
+/**
+ * Wrapped default formatting to take CET timezone into account
+ * should be used instead of normal format function from date-fns
+ *
+ * @param date Date to format
+ * @param formatStr Format string
+ * @param options options with if time zone should not be considered
+ * @returns formatted date
+ */
+function format(
+  date: string | number | Date,
+  formatStr: string,
+  options?: FormatOptions & {ignoreTimeZone?: boolean},
+) {
+  if (options?.ignoreTimeZone) {
+    return fnsFormat(date, formatStr, options);
+  } else {
+    return formatInTimeZone(date, CET, formatStr, {
+      ...(options as FormatOptionsWithTZ),
```

Comment:
Since `timeZone` is added right after `options` here, and the type name is "WithTZ", maybe more intuitive like this?
```ts
{
   ...options,
   timeZone: CET,
} as FormatOptionsWithTZ
```

Reply:
Agreed! Fixed